src/utils/sentry.ts

```diff
@@ -0,0 +1,39 @@
+import * as Sentry from "@sentry/node";
+import { IncomingMessage } from "http";
+import { NextApiRequest } from "next";
+
+// Helper to avoid duplicating the `init` call in every `/pages/api` file.
+// Also used in `pages/_app` for the client side, which automatically applies it for all frontend pages.
+if (process.env.NEXT_PUBLIC_SENTRY_DSN) {
+  Sentry.init({
+    dsn: process.env.NEXT_PUBLIC_SENTRY_DSN,
+    enabled: process.env.NODE_ENV !== "test",
+  });
+}
+
+Sentry.configureScope((scope) => {
+  scope.setTag("Node environment", process.env.NODE_ENV);
+});
+
+export const configureReq = (
+  request: NextApiRequest | IncomingMessage,
+): void => {
+  const headers = Object.entries(request.headers).reduce(
+    (acc, [key, value]) => (key !== "cookie" ? { ...acc, [key]: value } : acc),
+    {},
+  );
+
+  Sentry.configureScope((scope) => {
+    scope.setTag("url", request.url ? request.url : "");
+    scope.setTag("method", request.method ? request.method : "");
+
+    if ("query" in request) {
+      scope.setContext("query", request.query);
+      scope.setContext("body", request.body);
```

Comment:
Hmm, I need to think about it (and probably discuss it just to be sure).

Reply:
I'm not sure that it is correct, but if the callback api page throws when receiving the redirection from connect (with the `auth_code` as payload), sentry will catch the jwt, right? (from the query string or the body)

Reply:
from the query string no, since there will only ever be the authorization code. They would need to catch the data exchanged during a `fetch` call inside the api request. Which shouldn't be appearing, I guess